styles.css

```diff
@@ -364,6 +373,77 @@ a.logo {
   animation: rotation 2s 1;
 }
 
+.popSection {
+  position: fixed;
+  z-index: 10;
+  top: 0;
+  left: 0;
+}
+
+#popup-window {
+  z-index: 1;
+  background: #979493;
+  padding-top: 60px;
+  padding-left: 16px;
+  padding-right: 16px;
+  overflow-y: auto;
+  height: 100%;
+  position: fixed;
+}
+
+#popup {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  padding: 16px;
+}
```

Comment:
- [ ] Having made a great effort, I must commend you that you have done wonderfully well, 👍 👍  but I noticed your pop-up card in the `mobile version` are not well aligned, the card is hitting the wall to the bottom. I would kindly give this card some `margin bottom`.

- [ ] Secondly, I noticed you gave the cards container a `background color` that affects the main page from showing as seen in the Figma design below. When the pop-up modal card is displayed, the main page should still be showing underneath but in a Blur manner. You can do this by giving your card container a CSS property called `backdrop filter` with a value of `blur(8px)

> The card display well on the `desktop screen`......Kudos 👍 👍 

Your mobile Pop-up design 👇 

![lor-pp](https://user-images.githubusercontent.com/78634841/209413977-81651123-c708-45ec-9bf0-fead04ca9224.png)

Figma design 👇 

![pp2-mob](https://user-images.githubusercontent.com/78634841/209414025-52c6f521-6712-45b3-a8df-4a33e1b3a826.png)